pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[project]
+name = "gfw-data-api"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = "~= 3.10"
+dependencies = [
+    "aenum",
+    "aiofiles",
+    "aiohttp",
+    "alembic",
+    "arq",
+    "asgi_lifespan",
+    "async-lru",
+    "asyncpg",
+    "boto3",
+    "botocore",
+    "email-validator",
+    "fastapi",
+    "geoalchemy2<0.12",
+    "geojson",
+    "gino",
+    "gino_starlette",
+    "google-cloud-storage",
+    "httpcore",
+    "httpx",
+    "httpx-auth",
+    "newrelic",
+    "numpy<2",
+    "openapi_spec_validator",
+    "orjson",
+    "packaging",
+    "pendulum<3",
+    "pglast<2",
+    "psutil",
+    "psycopg2",
+    "pydantic<2",
+    "pyproj",
+    "python-multipart",
+    "retrying",
+    "shapely",
+    "sqlalchemy<1.4",
+    "sqlalchemy-utils",
+    "starlette",
+    "typer",
+    "uvicorn[standard]",
+]
+
+[dependency-groups]
+dev = [
+    "docker",
+    "geopandas",
```

Comment:
You dropped some comments from the Pipfile for geopandas, gfw_pixetl, pandas, and retrying.  Maybe add those comments back her in this file?

Reply:
Good point, will restore them!